packages/botonic-react/src/components/multichannel/multichannel-button.jsx

```diff
@@ -3,7 +3,7 @@ import React, { useContext } from 'react'
 
 import { RequestContext } from '../../contexts'
 import { truncateText } from '../../util'
-import { Button } from '../button'
+import { Button } from '../button/index'
```

Comment:
why we need to do import from /index? this should not be necessary

Reply:
I agree, I think there's no need to add the`/index` in the import path.

Reply:
removed